### ISSUE TITLE
About the Pix2pix

### ISSUE THREAD

**Issue by hello-trouble** [junyanz/pytorch-CycleGAN-and-pix2pix#1336]
Dear professor:
Thank you for your excellent paper and project. I am very insterested in the Pix2Pix network.  Can I set the weight of L1_loss and gan_loss to be 1 and 0.01 in the training process ? Besides, could the tanh() in the last of the generator  be removed or substituted by some other normalization operation? for example, to the range [-2,2] ?  looking forward to your help.

**Comment by junyanz** [comment]
Sure. You can do that. But you can also just reduce the learning rate. It is up to you. 
You can replace the tanh() with other layers or just remove it. 
If your input data range is [-2, 2], you could normalize it into [-1, 1] in the data loader. You need to modify the line [here](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/data/base_dataset.py#L111). You might also want to modify the tensor2im [function](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/361f8b00d671b66db752e66493b630be8bc7d67b/util/util.py#L24) which is used for saving images.

**Comment by hello-trouble** [comment]
Thank you for your kindly reply.
